### ISSUE TITLE
Unpic Svelte transformer (et al) is added as an attribute in the dom

### ISSUE THREAD

**Issue by ghostdevv** [ascorbic/unpic-img#697]
**Describe the bug**

When you have a custom transformer, it gets put in the dom which isn't ideal. In the case of svelte-cloudinary v2 this is quite large:

![image](https://github.com/user-attachments/assets/e4f77b23-a548-494f-a578-757c844747ac)

It happens with any prop you pass because of the `{...$$props}`.

**To Reproduce**

Use unpic svelte with a custom transformer, and then check the dom:

```svelte
<script>
	import { Image } from '@unpic/svelte';
</script>

<Image
	width={400}
	height={400}
	src=""
	transformer={() => {
		return 'https://res.cloudinary.com/svelte-cloudinary/image/upload/c_fill,w_400,h_400,g_center/f_auto/q_auto/v1/images/turtle?_a=BBEAACDM0}';
	}}>
</Image>
```

![image](https://github.com/user-attachments/assets/b44d2ca7-13b2-4efb-81ac-793fff09d7c4)

**Expected behavior**

In an ideal world anything that isn't an `img` prop wouldn't be added, but it's not too realistic to achieve. I think at the very least we should strip the `transformer` prop as it's quite an easy one, there might be others you think make sense to strip too!

**Comment by SkepticMystic** [comment]
Indeed, this happens with the `background` prop too, which can also get quite big.
shadcn-svelte handles this by destructuring all of their custom props, and capturing the `...rest`
